docs/reference/commandline/tag.md

```diff
@@ -16,5 +16,11 @@ parent = "smn_cli"
 
       --help               Print usage
 
+Image name must match the regular expression `[a-z0-9]+`. 
```

Comment:
The tag reference currently documents the command as:

```docker tag [OPTIONS] IMAGE[:TAG] [REGISTRYHOST/][USERNAME/]NAME[:TAG]```

Why do we specifically mention `USERNAME` component in the 4th argument and not in the 3rd? Unless needed for specific reasons, IMHO, it is a bit confusing as username is part of the image name. Shouldn't it be:

```docker tag [OPTIONS] [USERNAME/]NAME[:TAG] [REGISTRYHOST/][USERNAME/]NAME[:TAG]```

or

```docker tag [OPTIONS] IMAGE[:TAG] [REGISTRYHOST/]IMAGE[:TAG]```

I would vote for the first as it more explicit and clear. I can then document the regexes for all components.

On a side note, `REGISTRYHOST` can be specified in the 3rd argument as well. Note sure if this should be flagged as a CLI parser error.